pages/message-sent/index.tsx

```diff
@@ -4,6 +4,7 @@ import getConfig from "next/config";
 import { BaseContainer } from "@components/Page/BaseContainer";
 import { BetterSocialEventTracking } from "analytics/analyticsEventTracking";
 import { Helmet } from "react-helmet";
+import { MessageEnum } from "@shared/enum";
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `MessageEnum` is added but it doesn't seem to be used anywhere in the provided code. If it's not used, it should be removed to keep the code clean and maintainable.
```diff
- import { MessageEnum } from "@shared/enum";
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->